sdb/command.py

```diff
@@ -711,7 +712,8 @@ def caller(self, objs: Iterable[drgn.Object]) -> Iterable[drgn.Object]:
             # error
             raise CommandError(
                 self.name, 'no handler for input of type {}'.format(i.type_))
-        if not has_input:
+        if self.isfirst:
+            assert not has_input
```

Comment:
I think it would be more clear to structure this code like so, in which case we can remove the `has_input` variable:

```
if self.isfirst:
  assert that objs is empty
  yield from...
else:
  .. set up out_type ..
  .. set up baked ..
  for i in objs:
    ...
```

or
```
if self.isfirst:
  assert that objs is empty
  yield from...
  return
... set up out_type...
etc
```